src/rev_ai/apiclient.py

```diff
@@ -60,16 +48,21 @@ def submit_job_url(
 
         :param media_url: web location of the media file
         :param metadata: info to associate with the transcription job
-        :param callback_url: callback url to invoke on job completion as a webhook
-        :param skip_diarization: should rev.ai skip diaization when transcribing this file
-        :param skip_punctuation: should rev.ai skip punctuation when transcribing this file
-        :param speaker_channels_count: the number of speaker channels in the audio. If provided
-            the given audio will have each channel transcribed separately and each channel
-            will be treated as a single speaker. Valid values are integers 1-8 inclusive.
-        :param custom_vocabularies: a collection of phrase dictionaries. Including custom
-            vocabulary will inform and bias the speech recognition to find those phrases.
-            Each dictionary should consist of a key "phrases" which maps to a list of strings,
-            each of which represents a phrase you would like the speech recognition to bias
+        :param callback_url: callback url to invoke on job completion as
```

Comment:
was flake complaining about these lines?

Reply:
I think this change hurts readability.

https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds

Reply:
at least for this line, with `a webhook` dangling below. I'm neutral with the changes to the other docstrings below.